cmd/dbbench/main.go

```diff
@@ -164,8 +164,7 @@ func main() {
 		*threads = *iter
 	}
 
-	// Use built-in benchmarks.
-	benchmarks := bencher.Benchmarks()
+	benchmarks := []benchmark.Benchmark{}
```

Comment:
Reverted to old way of doing it.

I initially changed this as some databases without default benchmarks panic, and this would not allow these to run at all even with additional scripts supplied via flags. Reverted since its unrelated to the branch at hand.